### ISSUE TITLE
POI String Translation Issue: Duplicate String

### ISSUE THREAD

**Issue by charlescurley** [navit-gps/navit#556]
I am doing some work on the points of interest (POI) window under GTK. In the upper left corner of that window, there is a prompt for the text box for entering the search distance. It normally reads, "Select a distance to look for (km)". I was not happy with that, so I entered a "translation" in en.po.in. It now reads:

```
msgid "Select a distance to look for (km)"
msgstr "Select a search radius from screen center"
```

When I first open the POI window, I get the translated string. So far so good. But when I click on a category, the prompt changes to "Select a distance to look for (km)". OK, fine. I changed that translation as well:

```
msgid "Distance from screen center (km)"
msgstr "Select a search radius from screen center"

```
So, my question is, do we need two different strings for the same prompt? I think I can take one out of the source code with a simple change, but I'm not sure what the effect on the translations would be.

Relevant code is in gui_gtk_poi.c

**Comment by pgrandin** [comment]
Keep in mind that most of the code was written by people for whom English is not their native language. So there might be a few odd sentence constructions, or some sentences that are poorly formulated.

Feel free to submit a pull request to address that : i mean changing the source string itself. This will require an update of all translations, so we try to avoid doing it when possible, but sometimes it might be necessary. 

In the example you described, it might make sense to merge the source strings indeed.

Thanks!

**Comment by charlescurley** [comment]
Fixed in the course of pull request #638. Closing.
